apps/frontend-manage/src/pages/sessions/[id]/evaluation.tsx

```diff
@@ -195,6 +207,24 @@ function Evaluation() {
     statistics: {},
     status: SessionBlockStatus.Executed,
   })
+  const [textSize, setTextSize] = useState<'sm' | 'md' | 'lg' | 'xl'>('md')
+
+  const increaseTextSize = () => {
```

Comment:
Could be a case for useReducer -> cleaner state-based solution

Reply:
Logic has been replaced by extensive useReducer logic